src/index.ts

```diff
@@ -211,10 +211,12 @@ async function main(
 
   emitter.on('completed', () => {
     console.log(picocolor.green(`🎉 ${picocolor.bold('Copied project files')}`))
-    console.log(
-      picocolor.gray('Get started with:'),
-      picocolor.bold(`cd ${target}`),
-    )
+    if (target !== '.') {
```

Comment:
Is only `.` okay? I think `./` has the same problem.

Reply:
You're right. I'll add this and other possible ones too

Reply:
Thanks

Reply:
Please ping me after that.

Reply:
@yusukebe Can you check the updated code?